tests/lib/rules/no-unused-vars.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @fileoverview Tests for rule no-unused-vars.
+ * @author Pig Fang <g-plane@hotmail.com>
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/no-unused-vars");
+const RuleTester = require("eslint").RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+
+ruleTester.run("no-unused-vars", rule, {
+    valid: [
+        {
+            code: "import 'm'",
+            parserOptions: { sourceType: "module" }
```

Comment:
it is required to use `{ecmaVersion: 6}` in eslint v6 (not published yet).

also, you can use `const ruleTester = new RuleTester({parserOpitons: {...}});`